server/server.go

```diff
@@ -198,7 +199,7 @@ func (s *Server) startServer() error {
 	metadataGauge.WithLabelValues(fmt.Sprintf("cluster%d", s.clusterID)).Set(0)
 
 	s.rootPath = path.Join(pdRootPath, strconv.FormatUint(s.clusterID, 10))
-	s.leaderValue = s.marshalLeader()
+	s.member, s.leaderValue = s.memberInfo()
```

Comment:
Is `s.member` for leader or for current PD? seems `memberInfo` returns the leader?

Confusing...

Reply:
Current PD's member information. It will be saved to leader key if the PD successfully elected as leader.

Reply:
em, I think using memberValue may be more sensible. For me, leaderValue seems the value from the leader and the leader may be another node...

Reply:
good point.